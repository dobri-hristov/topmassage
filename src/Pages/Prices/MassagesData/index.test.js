import React from 'react'
import TestRenderer from 'react-test-renderer'
import PricesData from './index.js'

jest.mock('react-router-dom', () => ({
    Link: 'Link'
}))

describe('Prices page/PricesData component', () => {
    test('should render correctly with no data', () => {
        const component = TestRenderer.create(<PricesData />)
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
    test('should render correctly with data', () => {
        const component = TestRenderer.create(
            <PricesData path='classic' massage='класически' price='50' id='1' />
        )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})