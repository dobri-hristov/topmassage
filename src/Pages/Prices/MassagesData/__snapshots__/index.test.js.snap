// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prices page/PricesData component should render correctly with data 1`] = `
<article
  className="tr trOdd"
>
  <p
    className="linkName"
  >
    <Link
      className="link"
      to="massages/classic"
    >
      класически
    </Link>
  </p>
  <p>
    50
     лв.
  </p>
</article>
`;

exports[`Prices page/PricesData component should render correctly with no data 1`] = `
<article
  className="tr trOdd"
>
  <p
    className="linkName"
  >
    <Link
      className="link"
      to="massages/undefined"
    />
  </p>
  <p>
     лв.
  </p>
</article>
`;
